### PR TITLE
Add aws-ipi-baselinecaps from QE to cluster-bot workflow-launch

### PR DESCRIPTION
**Pull request by XiyunZhao**
- Add "cucushift-installer-rehearse-aws-ipi-baselinecaps-none" profile from QE to cluster-bot workflow-launch
- Make debugging the CI failure easier.

### CODE REVIEW

**Comment by XiyunZhao** [comment]
@liangxia @jhou1  could you help to take a look if this change is acceptable? thanks.
cc: @yapei

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 1]
```diff
@@ -79,6 +79,8 @@ workflows:
     platform: aws
   cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8:
     platform: aws
+  cucushift-installer-rehearse-aws-ipi-baselinecaps-none:
```
Comment (liangxia) [inline core-services/ci-chat-bot/workflows-config.yaml:82]: Would you mind to sort them with alphanumeric order ?

Comment (XiyunZhao) [reply core-services/ci-chat-bot/workflows-config.yaml]: Resloved

**Pull request merged**